tests/rptest/services/admin.py

```diff
@@ -868,3 +868,11 @@ def cloud_storage_usage(self) -> int:
         return int(
             self._request(
                 "GET", "debug/cloud_storage_usage?retries_allowed=10").json())
+
+    def refresh_node_health_info(self, node: Optional[ClusterNode] = None):
```

Comment:
Now you refresh only disk state. Name of request does not match with logic a little bit from my point of view.  Maybe rename it or refresh all health state?

Reply:
done.